e2e/framework/manager/rook_test_infra.go

```diff
@@ -364,6 +376,20 @@ func (r *rookTestInfraManager) InstallRook(tag string) (err error, client contra
 	if err != nil {
 		panic(err)
 	}
+	err = r.copyImageToNode(r.getContainerIdByName(k8s_master_container_name), rookClientTag)
```

Comment:
can you add the panic inside the `copyImageToNode` func? Then the func doesn't need to return an error and it will be cleaner